f3.go

```diff
@@ -68,15 +68,11 @@ func New(_ctx context.Context, manifest manifest.ManifestProvider, ds datastore.
 	}, nil
 }
 
-// SubscribeForMessagesToSign is used to subscribe to the message broadcast channel.
-// After perparing inputs and signing over them, Broadcast should be called.
-//
-// If the passed channel is full at any point, it will be dropped from subscription and closed.
-// To stop subscribing, either the closer function can be used, or the channel can be abandoned.
-// Passing a channel multiple times to the Subscribe function will result in a panic.
-func (m *F3) SubscribeForMessagesToSign(ch chan<- *gpbft.MessageBuilder) (closer func()) {
-	_, closer = m.busBroadcast.Subscribe(ch)
-	return closer
+// MessageStoSign returns a channel of outbound messages that need to be signed by the client(s).
+// - The same channel is shared between all callers and will never be closed.
```

Comment:
Sharing this channel is bad (you don't know which messages you will get) but I don think we need to protect against it.

Reply:
I think it's fine if it's a part of a documented part of the API.